personalize.module

```diff
@@ -1129,30 +1129,33 @@ function personalize_verify_agent($agent_name, $show_messages = TRUE) {
  * @param stdClass $agent_data
  *   (Optional) When provided then agent-specific transition steps can be
  *   included.  This is used for the "Scheduled" status.
+ * @param bool $include_all
+ *   (Optional) Include all transitions that are available.  If FALSE, only
+ *   those transitions that should be shown outside the campaign details
+ *   are returned.
```

Comment:
Hmm, I'm finding this parameter hard to wrap my head around. It's not clear to me why I would *not* want to include all. I had to read the description a few times to understand what it was for. Maybe we just need to switch it to an $exclude_[some type] param that defaults to true... at least if we can succinctly encapsulate in a variable name the types of transitions we'd want to exclude here. And if not, then this logic might not belong here but in the calling code?